scripts/gceworker.sh

```diff
@@ -98,6 +111,12 @@ case "${cmd}" in
     # Retry while vm and sshd start up.
     start_and_wait "${NAME}"
 
+    # Copy the ssh keys to the gceworker, this is needed to clone the managed-service repo
+    # Note: ssh-keyscan is needed to avoid the "Host key verification failed" error 
+    # when cloning the managed-service repo non-interactively
+    gcloud compute ssh "${NAME}" --command="mkdir -p ~/.ssh && ssh-keyscan github.com >> ~/.ssh/known_hosts"
+    gcloud compute scp "${SSH_KEY_FILE_PATH}" "${NAME}:~/.ssh/"
```

Comment:
Don't you additionally need the public key?

Reply:
No. AFAIK, public key is not really required in gceworker to clone or push.